src/qml/FeatureForm.qml

```diff
@@ -95,7 +95,7 @@ Page {
         }
 
         Repeater {
-          model: form.model && form.model.hasTabs ? form.model : 0
+          model: form.model.hasTabs ? form.model : 0
```

Comment:
are you sure of this one?

Reply:
When I follow the QML code that leads here, it seems there is (or at least should be) no case where `form.model` is falsy and different from an `AtttributeFormModel` instance.

This line does not particullary cause any warning, but I updated it together with 193, so there are no pieces of code checking for `form.mode` existance and tricking the reader to believe it can be falsy.